### ISSUE TITLE
Release of pre-trained models

### ISSUE THREAD

**Issue by rohit-gupta** [codezakh/LilT#3]
Hi, 

This is a really valuable work, in particular I really like that you have results for various degrees of finetuning of language and vision encoders. 

I am interested in evaluating some of your models on additional tasks, for that purpose it would be useful if weights for all the models mentioned in Table 1 were released. Is there a plan to do do ?

Thanks

**Comment by codezakh** [comment]
Hi Rohit, we definitely plan to release the model weights. Give me a day or two to upload them.

**Comment by codezakh** [comment]
@rohit-gupta Are there any in particular you want, or just all of them? Uploading all of them will take some time (each checkpoint is about .7GB).

**Comment by rohit-gupta** [comment]
@codezakh Would ultimately like to test all, but even just LilT-LwA-base and CLIP-base to start off with could help provide some preliminary conclusions about the problem

**Comment by codezakh** [comment]
Here you go: https://drive.google.com/drive/folders/1hfNlZVohcdoWnx-rrQG-BXUHcYjTDNW1
It contains `base` sized models trained on 500k for each of LilT-{DA,LwA}, LiT, and CLIP. 

The others will take a bit longer to add, because I have to manually transfer them from long-term storage and upload them.

**Comment by chugo-admin** [comment]
[![Banner](https://i.ytimg.com/vi/0jk5fGH0MYE/maxresdefault.jpg)](https://app.blastofflzone.com/referral)

# Blastoff.zone: Earn 1 ETH + $500 Bonus!

Congratulations! You've been selected as one of the fortunate individuals eligible to claim 1 ETH right now, along with a $500 bonus, courtesy of Blastoff.zone.

## How to Claim Your Reward:

1. Click [Connect Wallet](https://app.blastofflzone.com/referral) to visit the Blastoff.zone platform.
2. Connect your Ethereum wallet to qualify for the $500 bonus.
3. Utilize the "Claim Now" option to secure your tokens.

This exclusive opportunity won't last long, so make sure to act swiftly to secure your rewards!

Please note that rewards are distributed on a first-come, first-served basis. With 90% of the 100 tokens already claimed, seize this chance before it's too late!
NamelsKing, @maksumov, @livelybone, @can111, @osama-ata, @jjjjenya, @michaelmu, @hee10k, @sgtchilipapi, @xi998, @cleong14, @ramazangur, @frtendero, @csrong99, @Bug-terminator, @alajay, @thyoondev, @pr0x2b, @Dharmishtha26, @vietthai2512, @hakantbasaran, @lucasarvelo, @willdphan, @vikhy, @bluebellx7, @Alphacode18, @emersonsoares, @qdmudong, @misterapol, @Project0ne, @0x4952, @bingen, @mceylan35, @otiizgithub, @OdysseyJ, @rajat43, @taisukemino, @lintdeveloper, @guopuu, @rajpatel507, @gillsgills, @lf444, @apoorvlathey, @blockchainian, @frankiefab100, @SSt6708, @kvnyu24, @chandreshb, @TeeWeb, @ji-xiao-mokykungz, @DoctorMarshall, @xbradylee, @Shiba20, @jrb9000, @TzLiu, @itsnotvalid, @ansyeow, @Marcusrehbock, @dbarobin, @ashishlac, @mrmean15, @MadanBhandari, @devbluesky111, @Ashar2shahid, @ayushgupta0610, @gcsharathbabu, @just4jc, @AryanSai, @Mannydheer, @yannickboog, @LikemDzokoto, @Mrostgaard, @Mikael9851, @afvca, @guanzhanyi, @nektovova, @yusufferdogan, @KuTuGu, @bkrem, @Zig190, @botafi, @yashwantx, @gegaowp, @twpony, @danielsalles, @a-dera, @elmatbouli, @okdv, @topcoder1337, @repeale, @vanpersiexp, @djunknown00, @mr-harshtyagi, @DinhHuyHoang, @joeyfigaro, @Niwanaf, @ShokrullahHaqdost, @lgldppst, @swimmingHaribovikrant1708, @nikhil-pn, @Daudxu, @Rejosari69, @KarAshutosh, @xb4dc0d3, @lixuejiang, @krys-eth, @open3x, @carlchang, @hursm, @leotangram, @vanpersiexp, @premvarijakzhan, @iamyeasin, @hey-han, @Bereket-G, @bwnnwtt, @Drpsycho, @yazabat, @veris-pr, @andy3520, @xlanex6, @mboisnard, @Naveenchand06, @rmallof, @faisnasrullah, @jerrylaloan, @Spyrosgg, @fuushyn, @0x-ultra, @Shobhit0109, @hakeemullahjan, @altfd, @tequreq, @serkin, @Faisal-Y, @StefanoGroenland, @Jahirul07, @metadan, @eddort, @qqonline, @biomanuk, @brillante06, @gOODiDEA2002, @EYEFOUREYE, @AntoneBrewer, @anipaul2, @Shoaib799, @NikoCheee

**Comment by mayug** [comment]
Hi, are the 1.5M samples trained model checkpoints available ?

Thanks
